examples/basic-otlp-http/src/main.rs

```diff
@@ -19,14 +22,44 @@ fn init_tracer() -> Result<sdktrace::Tracer, TraceError> {
         .install_batch(opentelemetry_sdk::runtime::Tokio)
 }
 
+fn init_metrics() -> metrics::Result<sdkmetrics::MeterProvider> {
+    let export_config = opentelemetry_otlp::ExportConfig {
+        endpoint: "http://localhost:4318/v1/metrics".to_string(),
+        ..opentelemetry_otlp::ExportConfig::default()
+    };
+    opentelemetry_otlp::new_pipeline()
+        .metrics(opentelemetry_sdk::runtime::Tokio)
+        .with_exporter(
+            opentelemetry_otlp::new_exporter()
+                .http()
```

Comment:
https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-otlp/src/lib.rs#L254 You can now remove the comment about lack of metric support.

Reply:
Thanks done.